### PR TITLE
what is GitHub

### PR DESCRIPTION
**Pull request by caro401**
Sorry about the random config file changes, I forgot to take a separate branch.

Words are pretty rough, but i think there's something to say there. Looking mostly for feedback on general organisation and structure at this stage - did I hit all the points you think are relevant (and not a load of other unrelated things), are they presented coherently in a sensible order?

Read it in situ here https://4-what-is-github.oratio.pages.dev/what-is/github/

### CODE REVIEW

File: src/content/docs/what-is/github.md
Comment (danwaterfield) [inline src/content/docs/what-is/github.md:1]: built!

**Comment by danwaterfield** [comment]
Main points good, would maybe just add a single sentence of "Github allows you to share code, suggest revisions, and roll back changes if required". Obviously not as clumsy as that, but 1 line intro before you begin

File: src/content/docs/what-is/github.md
Comment (caro401) [reply src/content/docs/what-is/github.md:1]: I'm not sure what change you're requesting? Should I use the word "built" somewhere?

File: src/content/docs/what-is/github.md [hunk 1]
```diff
@@ -2,21 +2,55 @@
 title: What is GitHub?
 description: GitHub is a place where open-source code is built and shared
 sidebar:
-    label: GitHub?
+  label: GitHub?
 ---
 
-[GitHub](https://github.com) is a website where open-source code is build and shared. Think of code here in the loosest sense - smallish data sets in text files or spreadsheets, your thesis in a Markdown file, anything that is sort of vaguely text-based can happily live on GitHub.
```
Comment (caro401) [inline src/content/docs/what-is/github.md:8]: ohhhhhh this typo! will fix

**Pull request merged**